recipes/qt/5.x.x/conanfile.py

```diff
@@ -1132,6 +1136,15 @@ def _create_plugin(pluginname, libname, type, requires):
 
         if self.options.qtxmlpatterns:
              _create_module("XmlPatterns", ["Network"])
+        
+        if self.options.qtactiveqt:
+            _create_module("AxBase", ["Gui", "Widgets"])
```

Comment:
```suggestion
            _create_module("AxBase", ["Gui", "Widgets"])
            self.cpp_info.components["qtAxBase"].includedirs = ["include", os.path.join("include", "ActiveQt")]
```

This worked for me.

Reply:
https://doc.qt.io/qt-5/qaxbase.html

> Header:  #include \<QAxBase\>

Reply:
Indeed https://github.com/qt/qtactiveqt/blob/5.15.2/src/activeqt/axshared.prx
Thanks!

Reply:
@prince-chrismc the documentation is fine, it's just that QAxBase (and others) include file is not in the same folder as usual.

Reply:
Reading documentation was never my strong suite :) Perfect we trust our Qt expert ❤️

Reply:
> Perfect we trust our Qt expert heart

I can't agree more! :D